### PR TITLE
Bug fix: biases used twice in conv1x1 in network_blas

### PR DESCRIPTION
**Pull request by frpays**
This bug existed since the port of BLAS.

### CODE REVIEW

**Comment by dubslow** [comment]
Basically, Batchnorm::OffsetMeans handles this already in weights initialization right?

**Comment by frpays** [comment]
We use the batch normalization process to add the biases. So the biases are
added to the batchnorm means at init.
